src/VersionFeedController.php

```diff
@@ -165,7 +165,7 @@ public function linkToPageRSSFeed()
         RSSFeed::linkToFeed(
             $this->owner->Link('changes'),
             sprintf(
-                _t('RSSHistory.SINGLEPAGEFEEDTITLE', 'Updates to %s page'),
+                _t(VersionFeed::class . '.SINGLEPAGEFEEDTITLE', 'Updates to %s page'),
```

Comment:
This syntax won't work _yet_. I think for now we should use FQCN strings instead.

Reference: https://github.com/silverstripe/silverstripe-framework/issues/7647